presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -191,6 +188,194 @@ This table can then be queried in Presto::
 
     SELECT * FROM hive.web.page_view;
 
+
+.. _tuning-pref-hive:
+
+Tuning
+-------
+
+The following configuration properties may have an impact on connector performance:
+
+``hive.assume-canonical-partition-keys``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Boolean``
+ * **Default value:** ``false``
+ * **Description:** Disable optimized metastore partition fetching for non-string partition keys. Setting this property allows to avoid ignoring data with non-canonical partition values.
+
+
+``hive.domain-compaction-threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Integer`` (at least ``1``)
+ * **Default value:** ``100``
+ * **Description:** Maximum number of ranges allowed in a tuple domain without compacting it. Higher value will cause more data fragmentation but allows to use row skipping feature when reading ORC data. Setting this value higher may have large impact on ``IN`` and ``OR`` clauses performance in scenarios making use of row skipping.
```

Comment:
A higher value will cause more data fragmentation but allow the use of the row skipping feature when reading ORC data. Increasing this value may have a large impact on ``IN`` and ``OR`` clause performance in scenarios making use of row skipping.


Those were some small language changes, but this could use some clarification in general:  What is a tuple domain?  Does increasing this value have a positive or negative impact on IN and OR clause performance?  and by performance do you mean query runtime?